services/workflows-service/src/rule-engine/rule-store.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import { RuleSet } from './core/types';
+
+export type RuleStoreServiceFindAllOptions = {
+  databaseId: string;
+  source: 'notion';
+};
+
+@Injectable()
+export class RuleStoreService {
```

Comment:
Maybe `RuleRepository`? Repository doesn't necessarily have to access your Postgres DB, it can access any DB/store, like Notion.

Reply:
Oh that just for having additional 'store'/'repository' its doesnt matter at all...
We might remove this class after merge to notion in order to fetch all relevant rules